MediaRecorder.html

```diff
@@ -326,11 +329,11 @@
             playable, but the combination of all the Blobs from a completed
             recording MUST be playable.</p>
 
-            <p> If any Track within the MediaStream is muted at any time (i.e.,
-            if its <code>muted</code> flag is set to <code>true</code>), the UA
-            MUST insert black frames or silence until the Track is unmuted, or
-            record nothing, depending on the value of
-            <a>ignoreMutedMedia</a>.</p>
+            <p> The UA MUST insert black frames or silence, or record nothing,
+            depending on the value of <a>ignoreMutedMedia</a>, for as long as
+            a given <var>track</var> within the MediaStream  is
+            <a href="https://www.w3.org/TR/mediacapture-streams/#widl-MediaStreamTrack-muted">
+            muted</a>. This behaviour only applies to the muted tracks.</p>
 
```

Comment:
`muted` is a read-only attribute, so the code would not work.  But more importantly, I think `ignoreMutedMedia` works on a per-track basis, IOW, if MR is recording, say, 3 tracks, and one of them becomes `muted`, then a) the other two keep recording unperturbed and b) what MR records for the `muted` track depends on `ignoreMutedMedia`.